services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -45,6 +47,42 @@ class GitLabProjectVersioningTest extends AsyncWordSpec with ScalaFutures with M
         }
       }
     }
+
+    "getFile" should {
+      "return file with 200 response" taggedAs Service in {
+        val project = TestProjectUtils.getDummyProject()
+        val path = Paths.get("test.md")
+        val version = Some(Version(gitLabConfig.defaultFileVersion))
+
+        when(
+          mockHttpClient.get(
+            s"${gitLabConfig.url}/projects/${project.repository}/repository/files/${URLEncoder.encode(path.toString, "UTF-8")}/raw",
+            Map("ref" -> version.map(el => el.value).getOrElse(gitLabConfig.defaultFileVersion)),
```

Comment:
It is really strange to make a variable that contains `Some(_)` and then call `.getOrElse`. If you do need a value anyway, why did you wrap it into `Some(_)` ? :)
Define `version` as 
`val version = Version(gitLabConfig.defaultFileVersion)`

Reply:
You're absolutely right! Thanks a lot! 👍
Done